packages/babel-plugin-proposal-destructuring-private/test/fixtures/typescript/.valid-preset-typescript/options.json

```diff
@@ -0,0 +1,4 @@
+{
+  "plugins": ["proposal-destructuring-private"],
+  "presets": ["typescript"]
```

Comment:
This test is disabled because `presets` always run _after_ `plugins`. The order is preferred for preset-env and stage plugins, but not for typescript preset since it should run before other es plugins.